### PR TITLE
Chaotic Bifibration, Identity Bifibration

### PR DESCRIPTION
**Pull request by TOTBWF**
# Description

This PR adds the chaotic and identity bifibrations, and proves some properties about them.

## Notes
The name "chaotic bifibration" is not standard, so suggestions are welcome!

### CODE REVIEW

File: src/Cat/Displayed/Instances/Identity.lagda.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+IdDTotal-is-iso : is-precat-iso IdDTotal
+IdDTotal-is-iso .is-precat-iso.has-is-ff =
+  is-iso→is-equiv (iso hom (λ _ → total-hom-path _ refl refl) (λ _ → refl))
+IdDTotal-is-iso .is-precat-iso.has-is-iso =
+  is-iso→is-equiv (iso fst (λ _ → refl) (λ _ → refl))
+```
+
+<!--
+  [TODO: Reed M, 19/04/2023] Show that this is a left/right unit to composition
+  of displayed categories once we have equivalence/isomorphism of displayed categories.
```
Comment (plt-amy) [inline src/Cat/Displayed/Instances/Identity.lagda.md:132]: is the stuff in [paths of displayed categories](https://1lab.dev/Cat.Displayed.Path.html#paths-of-displayed-categories) not up to par?

**Review by plt-amy — APPROVED** [review]

File: src/Cat/Displayed/Instances/Identity.lagda.md (hunk 1 above)
Comment (TOTBWF) [reply src/Cat/Displayed/Instances/Identity.lagda.md:132]: Forgot this existed! Will do this in a separate PR though.

**Comment by ncfavier** [comment]
> suggestions are welcome

I'd call it constant or trivial, but the latter is already taken by what seems like a special case of it.

**Comment by TOTBWF** [comment]
Constant is a much better name! I'll open an issue about it

**Pull request merged**